### ISSUE TITLE
Windows Store Version not supported

### ISSUE THREAD

**Issue by DiscoInfernum** [GoobyCorp/DOOMSaveManager#16]
Not a necessity or priority, but the windows store version through xbox gamepass just came out and uses a different ID. I checked one of the original forum posts and there has not been a response there since it was added. Thought I would add it here because the gamepass version is incredibly buggy and stops working for people almost at random and I found this tool while trying to swap over to steam.

**Comment by ndlanier** [comment]
The Xbox Gamepass/Windows Store version definitely has encrypted save files and different structure possibly. Will be very difficult to implement a transfer tool.

**Comment by GoobyCorp** [comment]
Yeah messing with the anti-tamper on Windows Store executables would be a chore.
